Dockerfile

```
@@ -48,7 +48,10 @@ CMD "/bin/bash"
 
 # Database container
 FROM pls_base AS pls_mysql
-RUN apt-get install -y mysql-client mysql-server
+RUN apt-get install -y libreadline4
+RUN apt-get install -y perl
+RUN apt-get install -y mysql-client
+RUN apt-get install -y mysql-server
```

Each `RUN` statement creates a new image in docker that persists until a manual cleanup is performed. So, let's unify all of this into one `RUN` statement to avoid wasting disk space.

```suggestion
RUN \
    apt-get install -y libreadline4 && \
    apt-get install -y perl && \
    apt-get install -y mysql-client && \
    apt-get install -y mysql-server
```